lib/controllers/base.js

```diff
@@ -55,18 +56,21 @@ Controller.prototype.route = function() {
 
   // NOTE: is there a better place to do this mapping?
   var pathOptions = endpoint.string;
-  if (app.name === 'restify') {
-    if (self.method === 'delete')
+  if (this.isRestify) {
+    if (self.method === 'delete') {
```

Comment:
I will think about this, and come back with this issue in this evening, have a great day @mbroadst